7.3/Dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM php:7.3RC01-fpm
```

Comment:
This should be changed too

Reply:
But it does not matter when building specific versions